### PR TITLE
Feature/doc workflow schema

### PR DESCRIPTION
**Pull request by mathiaHT**
Add a documentation page for the workflow-definition.schema.json and add a small json to use for schema validation test

### CODE REVIEW

File: _includes/json_schema_type.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+{% if include.object.items['$ref'] %}
+
+{%- capture type -%}
+<a href="{{include.object.items['$ref'] | replace_first: '/definitions/', ''}}">
+array&lt;{{include.object.items['$ref'] }}&gt;
+</a>
+{% endcapture %}
+{{-type-}}
```
Comment (mathiaHT) [inline _includes/json_schema_type.html]: why do you first capture it and then directly print it?

File: _includes/json_schema.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+{% include json_schema_properties.html properties=schema.properties %}
+
+___
+
+{% for object in schema.definitions %}
+
+## {{object.first}}
+
+{% if object.last.title %}
```
Comment (mathiaHT) [inline _includes/json_schema.html]: `if object.last.title != undefined` ?

File: _includes/json_schema.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Title: {{object.last.title}}
+{% endif %}
+
+{% if object.last.type %}
```
Comment (mathiaHT) [inline _includes/json_schema.html]: `if object.last.type != undefined` ?

File: _includes/json_schema.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+Type: {% include json_schema_type.html object=object.last %}
+{% endif %}
+
+{% if object.last.pattern %}
```
Comment (mathiaHT) [inline _includes/json_schema.html]: `if object.last.pattern != undefined` ?

File: _includes/json_schema_type.html (hunk 1 above)
Comment (MarcAntoine-Arnaud) [reply _includes/json_schema_type.html]: you're right, I have changed that !

**Pull request merged**